packages/mobile/src/app/saga.ts

```diff
@@ -96,6 +102,37 @@ export function* appVersionSaga() {
   }
 }
 
+// Check the availability of Google Play Services. Log and report the result to.
+// Note: On iOS, this will always report a status of "IOS" and available = false.
+export function* checkGooglePlayServicesSaga() {
+  let result: GooglePlayServicesAvailability
```

Comment:
Looks like we should just skip this for iOS.

Reply:
Added a check to simply skip on iOS in 4c3a96e6fafaa559f6ef52608aa2114f8d293c96